GOVERNANCE.md

```diff
@@ -33,19 +33,3 @@ At the core group's discretion a new member may be added by a majority vote. Thi
  - Building supporting software and tooling
 
 Generally members are only removed at their own request are due to very long term inactivity. In extreme circumstances the core group can vote to remove a member.
-
```

Comment:
Since you moved this to a separate file, I think it should be linked to from this one